ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayKmlNetworkLinks/DisplayKMLNetworkLinks.qml

```diff
@@ -0,0 +1,63 @@
+// [WriteFile Name=DisplayKMLNetworkLinks, Category=Layers]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 1.4
+import QtQuick.Dialogs 1.2
+import QtQuick.Window 2.3
+import Esri.Samples 1.0
+
+DisplayKMLNetworkLinksSample {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property real scaleFactor: (Screen.logicalPixelDensity * 25.4) / (Qt.platform.os === "windows" || Qt.platform.os === "linux" ? 96 : 72)
+
+    MessageDialog {
+        id: messageDialog
+        title: "KML layer message"
+        text: currentKmlNetworkMessage
+    }
+
+    SceneView {
+        objectName: "sceneView"
+        anchors.fill: parent
+    }
+
+    Button {
```

Comment:
What I was originally picturing was something like a notification icon that would only show when there was some new coming in. Sort of like how slack has a little visual indicator that a new message has been received. Could you use the signal to flag this to disappear/reappear or show some visual indicator?